transcendence-app/src/user/decorators/api-file.decorator.ts

```diff
@@ -0,0 +1,17 @@
+import { ApiBody } from '@nestjs/swagger';
+
+export const ApiFile =
+  (fileName = 'file'): MethodDecorator =>
+  (target, propertyKey, descriptor) => {
+    ApiBody({
+      schema: {
+        type: 'object',
+        properties: {
+          [fileName]: {
+            type: 'string',
+            format: 'binary',
+          },
+        },
+      },
+    })(target, propertyKey, descriptor);
+  };
```

Comment:
https://stackoverflow.com/questions/65082784/how-do-i-annotate-an-endpoint-in-nestjs-for-openapi-that-takes-multipart-form-da